app/views/components/_page_preview.html.erb

```diff
@@ -5,22 +5,28 @@
 <% end %>
 
 <div class="app-c-preview">
+  <% mobile_preview = capture do %>
+    <%= tag.div class: "app-c-preview__mobile" do %>
```

Comment:
What's the benefit of using tag. here? I think we're using explicit HTML everywhere else.